### ISSUE TITLE
 Parser vs complex type with more than 5,000 nodes

### ISSUE THREAD

**Issue by Yadvo** [redhat-developer/vscode-xml#988]
I am working with a xsd that covers the Polish e-invoice:
http://crd.gov.pl/wzor/2023/06/29/12648/

Some of the elements have a weird approach as this:
<xsd:element name="NowySrodekTransportu" maxOccurs="10000">

There are some sample e-invoices available:
https://www.podatki.gov.pl/media/9538/przyk%C5%82adowe-pliki-dla-struktury-logicznej-e-faktury-fa-2.zip

While trying to validate them, I am getting this:
"message": "Current configuration of the parser doesn't allow the expansion of a content model for a complex type to contain more than 5,000 nodes.",

So I tried to do the following:
(1) -Djdk.xml.maxOccurLimit=11000
(2) in setting.json: "xml.symbols.maxItemsComputed": 11000,

The issue with the 5000 nodes still persists.

**Comment by angelozerr** [comment]
The settings should work but perhaps we have a bug

**Comment by cartercanedy** [comment]
I'm experiencing the same issue currently

**Comment by qwertycxz** [comment]
TL;DR

Add `-Djdk.xml.maxOccur=11000` in *vmargs*. Not `-Djdk.xml.maxOccurLimit`.
